scripts/start_runs_termination_script_in_background.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -eo pipefail
+set -x
+
+echo "Starting terminate_starting_and_started_runs in background to terminate orphaned ones."
+python /opt/dagster/app/scripts/terminate_starting_and_started_runs.py &
+
+# Hand off to the CMD
+exec "$@"
```

Comment:
Formatting: missing line at end of file.

Reply:
Fixed with https://github.com/mobidata-bw/ipl-dagster-pipeline/pull/187/commits/ed005b1dfa08b254b80d92c7cbb10a58b6bb4a3e